packages/js/core/src/types/Client.ts

```diff
@@ -49,6 +60,8 @@ export interface Client
   extends QueryHandler,
     SubscriptionHandler,
     InvokeHandler {
+  getApiCache(): ApiCache;
```

Comment:
I don't think we should expose the ApiCache. This is a sensitive part of the client that has yet to be fully fleshed out: https://github.com/polywrap/monorepo/issues/327

I feel like we should instead pass it internally to the resolvers directly. This way we don't have to worry about every-day users of the client having access to the cache. This is a better ownership relationship IMO (client owns cache, gives reference  resolvers [special permission]).

Will push a proposed change.